functions/zoom-downloader.py

```diff
@@ -341,12 +341,39 @@ def oc_series_found(self, ignore_schedule=False, override_series_id=None):
     @property
     def upload_message(self):
         if not hasattr(self, "self._upload_message"):
-            s3_filenames = {}
+            s3_files = {}
+            segment_durations = {}
             for file in self.recording_files:
-                if file.recording_type in s3_filenames:
-                    s3_filenames[file.recording_type].append(file.s3_filename)
+                segment = {
+                    "filename": file.s3_filename,
+                    "recording_start": file.file_data["recording_start"],
+                    "recording_end": file.file_data["recording_end"],
+                    "ffprobe_bytes": file.file_data["ffprobe_bytes"],
+                    "ffprobe_seconds": file.file_data["ffprobe_seconds"]
+                }
+                segment_durations[segment["recording_start"]] = segment["ffprobe_seconds"]
+                if file.recording_type in s3_files:
+                    s3_files[file.recording_type]["segments"].append(segment)
                 else:
-                    s3_filenames[file.recording_type] = [file.s3_filename]
+                    s3_files[file.recording_type] = {
+                        "segments": [segment]
+                    }
+
+            all_file_bytes = 0
+            all_file_seconds = 0
+            for view_data in s3_files.values():
+                view_data["view_bytes"] = sum(
+                    s["ffprobe_bytes"] for s in view_data["segments"]
+                )
+                view_data["view_seconds"] = sum(
+                    s["ffprobe_seconds"] for s in view_data["segments"]
+                )
+                all_file_bytes += view_data["view_bytes"]
+                all_file_seconds += view_data["view_seconds"]
+
+            recording_seconds = sum(
+                duration for duration in segment_durations.values()
+            )
 
```

Comment:
Calculating this number because it turns out the duration number we get from zoom seems to be wildy off in comparison to the ffprobe output.